src/mlpack/methods/ann/loss_functions/triplet_margin_loss.hpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file triplet_margin_loss.hpp
+ * @author Prince Gupta
+ *
+ * Definition of the Triplet Margin Loss function.
+ *
+ * For more information, refer the following paper.
+ *
+ * @code
+ * @article{Schroff2015,
+ *   author  = {Florian Schroff, Dmitry Kalenichenko, James Philbin},
+ *   title   = {FaceNet: A Unified Embedding for Face Recognition and Clustering},
+ *   year    = {2015},
+ *   url     = {https://arxiv.org/abs/1503.03832},
+ * }
+ * @endcode
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_ANN_LOSS_FUNCTION_TRIPLET_MARGIN_LOSS_HPP
+#define MLPACK_ANN_LOSS_FUNCTION_TRIPLET_MARGIN_LOSS_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+template <
+    typename InputDataType = arma::mat,
+    typename OutputDataType = arma::mat
+>
+class TripletMarginLoss
+{
+ public:
+  /**
+   * Create the TripletMarginLoss object with Hyperparameter margin.
+   * Hyperparameter margin defines the minimum numeric value by which distance
+   * between Anchor and Negative sample should be higher than the distance between
```

Comment:
Could you try doing the following : 

1. minimum value by which the distance between Anchor and Negative sample must exceed the distance between ...
This would increase readability.
Or if you come up with something that is better worded use that.